src/AppBundle/Controller/Pa/TeamController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Controller\Pa;
+
+use AppBundle\Entity as EntityDir;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use AppBundle\Controller\AbstractController;
+use AppBundle\Form as FormDir;
+
+/**
+ * @Route("/pa/team")
+ */
+class TeamController extends AbstractController
+{
+    /**
+     * @Route("/add-user", name="pa_team_user")
+     * @Template
+     */
+    public function addUserAction(Request $request)
+    {
+        $form = $this->createForm(new FormDir\User\UserDetailsPaType());
+
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $formData = $form->getData();
+
+            $this->getRestClient()->post('user/add', $formData);
```

Comment:
as you can see from the other "post" calls, it's normally a good practice to specify the JMS group you want to pass to the API. In some cases JMS might send more than expected, and either having the API failing for timeout, or saving data you didn't want to